client/blocks/plan-storage/style.scss

```diff
@@ -0,0 +1,38 @@
+
+.plan-storage__storage-label,
+.plan-storage__storage-link {
+	font-size: 11px;
+	display: inline-block;
+	height: 24px;
+	line-height: 24px;
+}
+
+.plan-storage__storage-label {
+	color: $gray-dark;
+	font-weight: 500;
+	margin-right: 10px;
+}
+
+.plan-storage__storage-link {
+	float: right;
```

Comment:
Speaking of which I'd recommend moving this class one layer down to the actual link instead of the span wrapper, so we can avoid a bare element rule